blocks/code/package.json

```diff
@@ -47,6 +45,7 @@
     "blockType": {
       "entryPoint": "react"
     },
+    "name": "@hash/code",
```

Comment:
Needed to pass `blockInfo={packageJSON.blockprotocol}` to `MockBlockDock` without type errors.